src/components/DatePicker/index.js

```diff
@@ -103,4 +104,14 @@ DatePicker.propTypes = propTypes;
 DatePicker.defaultProps = datePickerDefaultProps;
 DatePicker.displayName = 'DatePicker';
 
-export default withLocalize(DatePicker);
+const DatePickerWithRef = forwardRef((props, ref) => (
+    <DatePickerWithRef
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        forwardedRef={ref}
+    />
+));
+
+DatePickerWithRef.displayName = 'DatePickerWithRef';
```

Comment:
We can just use forwardRef directly on the DataPicker. No need to pass `forwardedRef`as a prop.

```js
function DatePicker({containerStyles, defaultValue, disabled, errorText, inputID, isSmallScreenWidth, label, maxDate, minDate, onInputChange, onTouched, placeholder, value}, ref) { ... }

export default React.forwardRef(DatePicker)
```

Reply:
Good point, we can consider refactoring other places where it's done that way.

Reply:
Actually we can't do it yet, because _forwardRef render functions do not support propTypes or defaultProps._ so we will be able to do it after TS migration.

Reply:
You don't need to set those for the forwarded component. You don't need to create a new component  either. Just directly export the forwarded one. The propTypes or defaultProps will be already set on the inner component itself.




```js
DatePicker.propTypes = propTypes;
DatePicker.defaultProps = datePickerDefaultProps;
DatePicker.displayName = 'DatePicker';

export default React.forwardRef(DatePicker);

```

Reply:
Actually you are right. Ignore my above comment.